cmd/minikube/cmd/start.go

```diff
@@ -1463,16 +1475,14 @@ func autoSetDriverOptions(cmd *cobra.Command, drvName string) (err error) {
 // validateKubernetesVersion ensures that the requested version is reasonable
 func validateKubernetesVersion(old *config.ClusterConfig) {
 	nvs, _ := semver.Make(strings.TrimPrefix(getKubernetesVersion(old), version.VersionPrefix))
+	oldestVersion := semver.MustParse(strings.TrimPrefix(constants.OldestKubernetesVersion, version.VersionPrefix))
+	defaultVersion := semver.MustParse(strings.TrimPrefix(constants.DefaultKubernetesVersion, version.VersionPrefix))
```

Comment:
You changed `oldestVersion, err := semver.Make` to `oldestVersion := semver.MustParse`, could you explain the significance of the change. Also, should `nvs, _ := semver.Make` be changed as well?

Reply:
sure the reason is are parsing a Constant that we know for sure it will parse so we can safely use Must Parse 
(previously we were doing a dummy error check for something that would NEVER be an err (parsing the default kubernetes version which was a forever constant and user cant change it)

so we get rid of 3 of these Dummy code blocks

```
		exit.Message(reason.InternalSemverParse, "Unable to parse oldest Kubernetes version from constants: {{.error}}", out.V{"error": err})
	}
```